python/tskit/trees.py

```diff
@@ -4740,6 +4744,19 @@ def write_vcf(
         on the VCF parser, and so we do **not** account for this change in
         coordinate system by default.
 
+        The ``allele_mapper`` argument allows the user to remap alleles in the
+        tree sequence (e.g., from integers to nucleotides). The value of
+        ``allele_mapper`` should be a function that takes a `.Variant` object
+        and returns a tuple of ``alleles, genotype_map``. The first should be
+        a list of unique alleles that will be output to the VCF file:
+        ``alleles[0]`` will be the REF allele and the remainder will be the ALT
+        alleles. The second, ``genotype_map``, is a numpy array of length equal
+        to the length of the number of alleles of the Variant object, and with
```

Comment:
delete "the length of", or maybe clearer as "is a numpy array of length equal to ``len(variant.alleles)``"